### PR TITLE
Avishag/add redux

### PR DESCRIPTION
**Pull request by avishag-gvili**

### CODE REVIEW

**Review by MiryamGabay — COMMENTED** [review]
add redux

File: client-side/src/redux/store.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { configureStore } from '@reduxjs/toolkit';
+import profileReducer from './profile/profile.slice.js';
+import VisitedWebsitesReducer from './visitedWebsites/visitedWebsites.slice.js';
+import userReducer from './user/user.slice.js';
+import preferenceReducer from './preference/preference.slice.js';
+import authReducer from './auth/auth.slice.js';
+import { useSelector } from 'react-redux';
```
Comment (SariGlick) [inline client-side/src/redux/store.jsx:7]: move it above the local imports

File: client-side/src/router/layout.jsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { Outlet } from "react-router-dom";
+import ResponsiveAppBar from "../stories/header/header";
+import React from "react";
```
Comment (SariGlick) [inline client-side/src/router/layout.jsx:3]: move it above the local imports

**Review by SariGlick — CHANGES_REQUESTED** [review]

**Review by SariGlick — APPROVED** [review]

**Review by Shoshana-Yudelevitz — COMMENTED** [review]
redux אבישג

**Review by SariGlick — APPROVED** [review]

**Review by e171271 — APPROVED** [review]

**Pull request merged**